package.json

```
@@ -24,9 +24,12 @@
   "author": "Scott Newcomer",
   "license": "MIT",
   "dependencies": {
+    "@embroider/macros": "^1.8.3",
     "ember-auto-import": "^2.2.3",
     "ember-cli-babel": "^7.26.6",
-    "ember-modifier": "^2.1.2 || ^3.0.0",
+    "ember-destroyable-polyfill": "^2.0.3",
+    "ember-modifier": "^3.2.7",
```

@And012 we should change this to

```suggestion
    "ember-modifier": "^2.1.2 || ^3.0.0 || ^4.0.0",
```